com.unity.uiwidgets/Runtime/engine/UIWidgetsPanel.cs

```diff
@@ -503,24 +535,31 @@ public partial class UIWidgetsPanel : IPointerDownHandler, IPointerUpHandler,
             if (_inputMode != UIWidgetsInputMode.Mouse) {
                 return;
             }
+
             D.assert(eventData.pointerId < 0);
             _isEntered = false;
             _wrapper.OnPointerLeave();
         }
 
         public void OnPointerDown(PointerEventData eventData) {
```

Comment:
we can wrap onPointerDown, onPointerUp, onDrag into one single #if block?

Reply:
I tried and failed. Since we inherent certain interface, wee need to add these function. Or do we add #if for interface as well?